src/components/CurratedContentCard/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { Image } from '../Image'
+import {
+  color,
+  radius,
+  space,
+  fontSize,
+  fontWeight,
+  device,
+  lineHeight,
+} from '../../theme'
+import { SmallText } from '../Text'
+import { transition } from '../../../src'
+
+const Container = styled.div`
+  background-color: ${color.elevatedBackground};
+  display: flex;
+  inline-size: 302px;
+  block-size: 314px;
+  flex-direction: column;
+  cursor: pointer;
+  border-radius: ${radius.lg};
+  overflow: hidden;
+
+  @media ${device.tablet} {
+    inline-size: 639px;
```

Comment:
Depending on if this is being used in a carousel or not, I don't think it's a good idea to give this a fixed size. If we would use this separately, in for example a full width container, it's not spanning the whole width which looks weird probably:

<img width="1086" alt="CleanShot 2023-05-05 at 12 53 38@2x" src="https://user-images.githubusercontent.com/14276144/236439896-dd52186f-4751-4c9f-b857-a5c4fe8561c1.png">

Reply:
yeah I asked John to give me aspect ratio insge

> I think this carousel looks similar to the `CardCarousel`. Should we get rid of that?

Yeah I asked - atm it is used once in sirius in the partner page. So thought it was out of scope for now and apparently this partner page will get a revamp soon as well